### ISSUE TITLE
PageItems empty when changing array contents

### ISSUE THREAD

**Issue by matrox2nz** [ErikSchierboom/knockout-paging#14]
I think I found a bug. I have setup knockout-paging with a simple observable array of items. I noticed when I am NOT on the _first_  page all the items suddenly disappear if I change the underlying array values. As a work-around I figured out that I had to reset the page using .toFirstPage() function and then it would work as intended.

**Comment by ErikSchierboom** [comment]
Thanks for logging this issue. Could you perhaps create a small repro?

**Comment by matrox2nz** [comment]
Sure, I will post when I get a chance. Great plugin btw!

**Comment by guilhermewaess** [comment]
Aw!! this happen when your pageItens is to low as your current page selected, for example:

``` js
target.pageSize(2);
target([1,2,3,4,5,6]);
target.pageNumber(3);
target.pageSize(6);

Current pageItems is empty.
```

[Here is a repro](http://jsbin.com/liqacu/edit?html,js,output)
For fast solution on my project, I just reset the page like @matrox2nz.
I think the solution is verify if current page exists, if not, go to last page.
I don't know if this is a final solution, but I would be happy to help analyze and solve :D

**Comment by ErikSchierboom** [comment]
If you'd like to fix this yourself, be my guest. If not, no worries! I'll look into it then. Thanks for the analysis so far!

**Comment by ErikSchierboom** [comment]
This bug has been fixed in version 0.3.2, which has just been released. Thanks a lot for the detailed information and the repro!
